File: examples/equos-python-integration/src/python_integration/list_brains.py
from python_integration.client import get_client


def list_brains() -> None:
    client = get_client()
    response = client.brains.list(take=50)

    print(f"{len(response.brains)} of {response.total} brain(s):")
    for b in response.brains:
        preview = " ".join(b.instructions.split())[:60]
        ellipsis = "…" if len(b.instructions) > 60 else ""
        print(f"- {b.id} · {preview}{ellipsis}")


if __name__ == "__main__":
    list_brains()

File: examples/equos-nodejs-integration/src/list-brains.ts
import { client } from "./client.js";

const { total, brains } = await client.brains.listBrains({ take: 50 });

console.log(`${brains.length} of ${total} brain(s):`);
for (const b of brains) {
  const preview = b.instructions.slice(0, 60).replace(/\s+/g, " ");
  console.log(`- ${b.id} · ${preview}${b.instructions.length > 60 ? "…" : ""}`);
}
